darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -915,11 +938,86 @@ def fit_from_dataset(
                 "best=False)`. Finally, train the model with `model.fit(..., epochs=new_epochs)` where "
                 "`new_epochs` is the sum of (epochs already trained + some additional epochs)."
             )
+        self.train_loader = train_loader
+        self.val_loader = val_loader
 
-        # Train model
-        self._train(train_loader, val_loader)
+        # Train model if not in "setup_only" mode
+        if not setup_only:
+            self._train(train_loader, val_loader)
         return self
 
+    def lr_find(
+        self,
+        min_lr: float = 1e-08,
```

Comment:
We could also just accept the `lr_find`parameters as kwargs (apart from those that we will set such as model, dataloaders, update_attr). Then we just link to the PyTorch Lightning for the `lr_find` docs.

Like that we can avoid having to maintain possible changes to the lr_find API.

Reply:
Cool. This makes things more transparent.